backend/prismaCalls.ts

```diff
@@ -42,4 +43,65 @@ export async function getPackageHistories(queryName: apiSchema.PackageName){
         console.log(`Error in getPackageHistories: ${error}`);
         return null;
     }
+}
+// For download endpoint
+export async function getPackageWithMetadataAndData(queryName: string, version: string): Promise<{ metadata: prismaSchema.PackageMetadata, data: prismaSchema.PackageData } | null> {
```

Comment:
I think it'd be better to just call this "getPackage"
Also, change it to returning a Promise of type package from the apiSchema.
and queryName should be of type apiSchema.PackageName (look at getMetaDataArray)